templates.dist/fields.php

```diff
@@ -122,6 +122,7 @@ function renderFormField(string $fieldName, onOffice\WPlugin\Form $pForm, bool $
 		$selectedValue = $pForm->getFieldValue($fieldName, true);
 		$isRangeValue = $pForm->isSearchcriteriaField($fieldName) && $searchCriteriaRange;
 		$fieldLabel = $pForm->getFieldLabel($fieldName, true);
+		$isHiddenField = $pForm->isHiddenField($fieldName);
```

Comment:
Please define the hidden attribute here 
using:
```
$isHiddenField = $pForm->isHiddenField($fieldName);
$hiddenAttribute = $isHiddenField ? 'class="oo-hidden-field" readonly ' : '';
```

instead of:
```
if ($isHiddenField === true) {
	$inputType .= 'class="oo-hidden-field" disabled ';
}
```